docs/README.md

```diff
@@ -12,7 +12,9 @@ This documentation explains how to use [ReVanced Manager](https://github.com/rev
    3. [🔄 Updating ReVanced Manager](2_3_updating.md)
    4. [⚙️ Configuring ReVanced Manager](2_4_settings.md)
 3. [❔ Troubleshooting](3_troubleshooting.md)
-4. [🔨 Building from source](4_building.md)
+
+## 👋 Developer version
+Interested in contributing? Consider checking out developer version of the [documentation](/developers/README.md).
```

Comment:
How about `...? Consider checking out the developer [documentation](/developers/README.md)`

Reply:
```suggestion
Interested in contributing? Consider checking out the developer [documentation](/developers/README.md)
```


Co-authored-by: Palm <palmpasuthorn@gmail.com>